augur/__init__.py

```diff
@@ -73,6 +73,9 @@ def run(argv):
     args = make_parser().parse_args(argv)
     try:
         return args.__command__.run(args)
+    except AugurError as e:
+        print(f"ERROR: {e}")
+        sys.exit(2)
```

Comment:
The `AugurError` handling builds on ideas in #914. Ideally, merge that PR first but merge conflicts can be fixed either way.